### ISSUE TITLE
resource android:attr/lStar not found.

### ISSUE THREAD

**Issue by admonxetrii** [endigo/flutter_pdfview#240]
Execution failed for task ':flutter_pdfview:verifyReleaseResources'.
> A failure occurred while executing com.android.build.gradle.tasks.VerifyLibraryResourcesTask$Action
   > Android resource linking failed
     ERROR: ..\build\flutter_pdfview\intermediates\merged_res\release\values\values.xml:132: AAPT: error: resource android:attr/lStar not found.

**Comment by devmgs** [comment]
Same here

* What went wrong:
Execution failed for task ':flutter_pdfview:verifyReleaseResources'.
> A failure occurred while executing com.android.build.gradle.tasks.VerifyLibraryResourcesTask$Action
   > Android resource linking failed
     ERROR:/Users/..../myFlutterApps_v2/...../build/flutter_pdfview/intermediates/merged_res/release/values/values.xml:132: AAPT: error: resource android:attr/lStar not found.

**Comment by JonssonViktor** [comment]
Same here:

> Execution failed for task ':flutter_pdfview:verifyReleaseResources'.
> > A failure occurred while executing com.android.build.gradle.tasks.VerifyLibraryResourcesTask$Action
>    > Android resource linking failed
>      ERROR:/Users/builder/clone/app/build/flutter_pdfview/intermediates/merged_res/release/values/values.xml:132: AAPT: error: resource android:attr/lStar not found.

**Comment by devmgs** [comment]
for the time being, it is not resolved to add this to pubspec
```
dependency_overrides:
  # Keep until https://github.com/endigo/flutter_pdfview/issues/240 is resolved.
  flutter_pdfview: 1.2.9
```

**Comment by chailirTokoplas** [comment]
i have same issues...

**Comment by dunjil** [comment]
I am having the same error when building the realease apk:
Execution failed for task ':flutter_pdfview:verifyReleaseResources'.
> A failure occurred while executing com.android.build.gradle.tasks.VerifyLibraryResourcesTask$Action
   > Android resource linking failed
     ERROR:/home/duna/.gradle/caches/transforms-3/edbb5dc2421b94be08914dd4aaebfe48/transformed/core-1.7.0/res/values/values.xml:105:5-114:25: AAPT: error: resource android:attr/lStar not found.

**Comment by JonssonViktor** [comment]
as mentioned above

dependency_overrides:
  # Keep until https://github.com/endigo/flutter_pdfview/issues/240 is resolved.
  flutter_pdfview: 1.2.9

> i have same issues...
> 
> ERROR:/Users/user/my_app/build/flutter_pdfview/intermediates/merged_res/release/values/values.xml:132: AAPT: error: resource android:attr/lStar not found.
> 
> any solution?

**Comment by MCYBA** [comment]
in my case, I downloaded version `1.2.9` from [here](https://pub.dev/packages/flutter_pdfview/versions). And I created my own repository and pushed the code [here](https://github.com/MCYBA/flutter_pdfview-1.2.9.git).
Then I placed it in pubspec.yaml like this:
```
dependencies:
  ...
  flutter_pdfview:
    git:
      url: https://github.com/MCYBA/flutter_pdfview-1.2.9.git
      ref: main
```

for now it's working.

**Comment by aqueebAiolos** [comment]
fixed it by adding flutter_pdfview: "1.2.2" to dependeices

**Comment by endigo** [comment]
Please follow this trick:
https://stackoverflow.com/a/69050529
